### DISCUSSION TITLE
CI pipeline stuck when adding a new element to matrix

### DISCUSSION THREAD

**Discussion by renardeinside** [community/community#25310] [Actions]
<p>Hi everyone.</p>
<p>In my project CI pipeline, I previously had a single pipeline running for one OS, now I’ve added another one. After it, I cannot close the PR due to a pretty strange status error:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/7/07a1a10b0111bb7bfb471669584a64ae563b5a04.png" data-download-href="/uploads/short-url/15vD7vYNWlSU28JpjF6MvPWu3c0.png?dl=1" title="Screenshot 2022-02-05 at 11.39.53" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/4979594/181090208-e4199af0-71f7-494f-a2d5-42e0707c5228.png" alt="Screenshot 2022-02-05 at 11.39.53" data-base62-sha1="15vD7vYNWlSU28JpjF6MvPWu3c0" width="690" height="307" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/7/07a1a10b0111bb7bfb471669584a64ae563b5a04_2_690x307.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/7/07a1a10b0111bb7bfb471669584a64ae563b5a04_2_1035x460.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/7/07a1a10b0111bb7bfb471669584a64ae563b5a04_2_1380x614.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/7/07a1a10b0111bb7bfb471669584a64ae563b5a04_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot 2022-02-05 at 11.39.53</span><span class="informations">1846×822 116 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>It seems like GitHub Actions expecting a renamed pipeline to be successfully finished. The PR could be found <a href="https://github.com/databrickslabs/dbx/pull/157" rel="noopener nofollow ugc">here</a>.</p>
<p>Has anyone run into the same issue? Any ideas on how to fix it?</p>

**Comment by airtower-luna** [comment]
<p>The required jobs are set in the branch protection rules, by name. If you rename a required job, you’ll need to update the branch protection settings to require the new job(s), instead.</p>

**Comment by renardeinside** [comment]
<p>hi <a class="mention" href="/airtower-luna">@airtower-luna</a> ,<br>
I haven’t renamed the job per se - it’s still <code>test-pipeline</code>, I just added a second branch of it to the matrix. For some unknown reason, now it has build/ prefix, which leads to this error.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I’m not sure why the prefix changed, but the solution remains the same: Update the branch protection rule to require the job as it is right now. Personally I prefer to have the workflow name in there, too, makes it easier to recognize the job.</p>
